ogr/ogrsf_frmts/geojson/ogrgeojsondriver.cpp

```diff
@@ -733,6 +733,7 @@ void RegisterOGRGeoJSON()
         "Integer64List RealList StringList Date DateTime");
     poDriver->SetMetadataItem(GDAL_DMD_CREATIONFIELDDATASUBTYPES, "Boolean");
     poDriver->SetMetadataItem(GDAL_DMD_SUPPORTED_SQL_DIALECTS, "OGRSQL SQLITE");
+    poDriver->SetMetadataItem(GDAL_DCAP_MEASURED_GEOMETRIES, "YES");
```

Comment:
Is this change related with the GeoJson FID issue?

Reply:
> Is this change related with the GeoJson FID issue?

no, that's why I put it in a separate commit. It is necessary for test_ogrsf to be happy